packages/core/src/artifact/ArtifactTypes.ts

```diff
@@ -5,6 +5,7 @@ export interface IArtifactTypePatterns {
 export const ArtifactTypePatterns: IArtifactTypePatterns = {
   BITBUCKET_FILE: /bitbucket\/file/,
   CUSTOM_OBJECT: /custom\/object/,
+  CLOUDRUN: /cloudrun\/(.*)/,
```

Comment:
Forgive me if I'm not familiar with Cloud Run. Why does it have its own artifact type if the manifest is just YAML/JSON?

Reply:
So based on back-end implementation CLOUDRUN type is not much useful for now like KUBERNETES, So removing it for now will add it if required in future based on enhancement request .